sgpt/handlers/repl_handler.py

```diff
@@ -30,6 +30,9 @@ def handle(self, prompt: str, **kwargs: Any) -> None:  # type: ignore
         )
         typer.secho(info_message, fg="yellow")
 
+        if prompt:
+           super().handle(prompt, **kwargs)
```

Comment:
In this case, if we strat REPL with `--shell` the first command suggestion will not be possible to execute.